portfolio.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Georgia Adam</title>
+  <link rel="shortcut icon" href="/assets/forest.png">
+  <link rel="stylesheet" href="/styles/style.css">
+  <link rel="stylesheet" href="/styles/portfolio.css">
+
+</head>
+
+<body>
+  <nav>
+    <ul>
+      <li><a href="/index.html"> Home </a></li>
+      <li><a href="/portfolio.html"> Portfolio </a></li>
+      <li><a href="/about.html"> About </a></li>
+    </ul>
+  </nav>
+  
+  <header>
+    <h1>Georgia Adam</h1>
+    <h2>Software Engineering Student @ Ada Developers Academy</h2>
+  </header>
+
+  <section id="blurb">
+    <h3>
+      Please find a selection of projects I have worked on below.
+    </h3>
+  </section>
```

Comment:
I would suggest including this `section` inside the `main` element below since it is the header for the dominant content of the `body`.